cidc_api/services/ingestion.py

```diff
@@ -287,6 +287,9 @@ def upload_manifest(
         session=session,
     )
 
+    # Manifest has been uploaded successfully
+    manifest_upload.alert_upload_success()
```

Comment:
nit: as this `alert_upload_success` in case of assay upload is called within a model method, I would consider moving this call here to the `manifest.create` - as it `commit`s the session and thus should issue that email in a consistent way with assay uploads.